frontends/web/src/routes/bitsurance/widget.tsx

```diff
@@ -99,18 +99,50 @@ export const BitsuranceWidget = ({ code }: TProps) => {
     current.contentWindow?.postMessage(message, '*');
   };
 
+  const sendAddressWithXPub = (address: string, sig: string, xpub: string) => {
```

Comment:
Maybe better to refactor `sendAddress` by adding an optional xpub parameter, to avoid duplicating the code. Do we plan to use the version without xpub at all? Otherwise, we could just remove that unused code.

Reply:
I will remove the old sendAddress.